Content.Server/Cargo/CargoOrderDataManager.cs

```diff
@@ -95,5 +96,10 @@ public List<CargoOrderData> GetOrdersFromAccount(int accountId)
                 return null;
             return account.GetOrders();
         }
+
+        public Tuple<int,int> GetCapacity()
+        {
+            return _components.FirstOrDefault().GetCapacity();
```

Comment:
You probably want to get the database here instead of just the first component.

The purpose of the databases is so that people can have separate space stations on one map with separate cargo shopping carts.

Reply:
Yeah, I was just working with what was available in the class already but I'll see what I can do